src/main/java/com/icthh/xm/uaa/domain/converter/RoleKeyConverter.java

```diff
@@ -0,0 +1,78 @@
+package com.icthh.xm.uaa.domain.converter;
+
+
+import static com.google.common.collect.Iterables.getFirst;
+import static java.util.Collections.emptyList;
+import static java.util.Optional.ofNullable;
+import static org.apache.commons.lang3.StringUtils.isNoneBlank;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.core.type.TypeReference;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.datatype.jsr310.JavaTimeModule;
+import com.icthh.xm.uaa.service.TenantPropertiesService;
+import java.io.IOException;
+import java.util.List;
+import javax.persistence.AttributeConverter;
+import javax.persistence.Converter;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+
+@Converter
+@Component
+@Slf4j
+public class RoleKeyConverter implements AttributeConverter<List<String>, String> {
+
+    private TenantPropertiesService tenantPropertiesService;
+    private final ObjectMapper mapper = new ObjectMapper();
+
+    public RoleKeyConverter(){
+        mapper.registerModule(new JavaTimeModule());
```

Comment:
you can write `private final ObjectMapper mapper = new ObjectMapper().registerModule(new JavaTimeModule());`

Reply:
changed